curl-sys/build.rs

```diff
@@ -287,8 +298,10 @@ fn main() {
             .define("USE_THREADS_WIN32", None)
             .define("HAVE_IOCTLSOCKET_FIONBIO", None)
             .define("USE_WINSOCK", None)
+            .file("curl/lib/bufref.c")
             .file("curl/lib/system_win32.c")
             .file("curl/lib/version_win32.c")
+            .file("curl/lib/vauth/digest_sspi.c")
```

Comment:
The name suggests that `digest_sspi` is `USE_WINDOWS_SSPI` specific feature. Should this be under [line 256](https://github.com/alexcrichton/curl-rust/pull/401/files#diff-4b9a5dbfe7ad27e69c2b850cb2b18e53643d175862f843a5f2d1194df83609a5L256) instead?

The same goes for `spnego_sspi` on [line 295](https://github.com/alexcrichton/curl-rust/pull/401/files#diff-4b9a5dbfe7ad27e69c2b850cb2b18e53643d175862f843a5f2d1194df83609a5L295) which already existed.